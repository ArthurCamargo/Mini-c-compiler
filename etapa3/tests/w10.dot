digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n1 -> n4;
n2;
n2 [label="3"];
n3;
n3 [label="a"];
n4;
n4 -> n5;
n4 -> n6;
n5;
n5 [label="a"];
n6;
n6 [label="2"];
n4 [label="<<"];
n1 [label="<="];
n0 [label="f"];

}