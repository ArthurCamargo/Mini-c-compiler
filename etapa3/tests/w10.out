0x55eb9a2bc350
0x55eb9a2bc350, 0x55eb9a2bc480
0x55eb9a2bc480
0x55eb9a2bc480, 0x55eb9a2bc430
0x55eb9a2bc480, 0x55eb9a2bc3c0
0x55eb9a2bc480, 0x55eb9a2bc560
0x55eb9a2bc430
0x55eb9a2bc430 [label="3"];
0x55eb9a2bc3c0
0x55eb9a2bc3c0 [label="a"];
0x55eb9a2bc560
0x55eb9a2bc560, 0x55eb9a2bc4f0
0x55eb9a2bc560, 0x55eb9a2bc5b0
0x55eb9a2bc4f0
0x55eb9a2bc4f0 [label="a"];
0x55eb9a2bc5b0
0x55eb9a2bc5b0 [label="2"];
0x55eb9a2bc560 [label="<<"];
0x55eb9a2bc480 [label="<="];
0x55eb9a2bc350 [label="f"];
